src/playercards/AllCardsBag.ttslua

```diff
@@ -1,8 +1,165 @@
 local cardIdIndex = { }
-local classAndLevelIndex = { }
+local otherIndex
+classIndex = { }
+levelIndex = { }
 local basicWeaknessList = { }
 local uniqueWeaknessList = { }
-local cycleIndex = { }
+cycleIndex = { }
+traitsIndex = { }
+willpowerIconsIndex = { }
+intellectIconsIndex = { }
+combatIconsIndex = { }
+agilityIconsIndex = { }
+wildIconsIndex = { }
+typeIndex = { }
+slotIndex = { }
+costIndex = { }
+permanentIndex = { }
+usesIndex = { }
+specialCriteriaIndex = { }
+
+local specialCriteriaManualOverride = {
```

Comment:
Now that we have 3 special criterias already (and might get more down the line), what do you think about including them in the metadata as a single `|` separated field (like `class`)?
Example:
`"specialCriteria": "healsHorror|parley"`

Reply:
Yeah...this is probably a good idea.  I will probably make a separate PR for that though, because it will be a _lot_ of metadata changes